ui/packages/catalog-ui-search/src/main/webapp/js/EventSourceUtil.js

```diff
@@ -0,0 +1,30 @@
+var EventSource = require('eventsource')
+
+const ORIGIN_HEADER = 'https://localhost:8993/'
```

Comment:
I know this shouldn't be hard-coded but I'm not sure how to get the right value from this. @bdeining any ideas?

Reply:
I think we need to use ` credentials: 'same-origin'` https://github.com/codice/ddf-ui/blob/master/ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/utils/fetch/fetch.tsx#L55

Reply:
i guess this `credentials` parameter isn't technically part of the headers - the [EventSource](https://github.com/EventSource/eventsource) object I think only takes a url and an object representing headers but I'll try it this way.

Reply:
looks like it worked 👍